pkg/apis/controller/v1beta1/appwrapper.go

```diff
@@ -246,7 +246,7 @@ type AppWrapperStatus struct {
 	// Microsecond level timestamp when controller first sees QueueJob (by Informer)
 	ControllerFirstTimestamp metav1.MicroTime `json:"controllerfirsttimestamp,omitempty"`
 
-	// Microsecond level timestamp when controller first sets appwrapper in state Running
+	// Microsecond level timestamp when controller first dispatches appwrapper
 	ControllerFirstDispatchTimestamp metav1.MicroTime `json:"controllerfirstdispatchtimestamp,omitempty"`
```

Comment:
That one could also be lower camel cased?

Reply:
Potentially, but I am not sure what tooling would break if we start changing existing fields.  @asm582 and @metalcycling can you chime in ?

I am weary of chasing more in this PR, it's big enough already.

Reply:
It wasn't there in the CRD, isn't it? So it can be considered as a new field?

Reply:
Can we do these field changes as a separate PR, please? In general, we have seen that capitals in field names make it a mandatory field.

Reply:
That's about the name of the JSON field, suggesting to follow the Kubernetes API convention, i.e., lower camel cased names. It's a new field added to the CRD, so that's the best place to follow the convention. Another option is to not add the field in that PR, and add it in a separate PR.

Reply:
I see thanks, I vote for adding it as part of separate PR once this merged

Reply:
Will do.